### PR TITLE
Added Filter category function

### PR DESCRIPTION
**Pull request by kanbi18**
First filtering function by category

### CODE REVIEW

File: maptivist/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.google.sps</groupId>
-  <artifactId>maptivist-step-2020</artifactId>
+  <artifactId>maptivist</artifactId>
```
Comment (flying-bus) [inline maptivist/pom.xml:7]: Why is this changing?

**Comment by freeboson** [comment]
Merge in the latest master, and then ping us for review.

**Comment by kanbi18** [comment]
I want to make some adjustments to tests.

File: maptivist/src/main/java/com/google/sps/servlets/MarkerServlet.java [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+            Set<String> links = createLinkObject((String) entity.getProperty("links"));
+            ArrayList<String> flags = createFlagObject((String) entity.getProperty("flags"));
+        
+            Marker marker = new Marker(title, description, latitude, longitude, links, categories);
+            markers.add(marker);
+        }
+        return markers;
+    }
+ 
+    private Boolean validateInput(String title, String longitude, String latitude, String description, String links) {
```
Comment (TBarrett553) [inline maptivist/src/main/java/com/google/sps/servlets/MarkerServlet.java:111]: The Marker classes constructor is exactly the same as this validate input method, so I don't think this is necessary. The marker can't be made if specific components are empty.

**Pull request closed without merging**